contracts/veBalFeeInjector.sol

```diff
@@ -19,30 +20,33 @@ import "./interfaces.sol";
  * @notice There are a number of management functions to allow the owner to sweep tokens and/or change how things work.
  */
 contract veBalFeeInjector is ConfirmedOwner, Pausable {
-  event keeperRegistryUpdated(address oldAddress, address newAddress);
+  event KeeperRegistryUpdated(address oldAddress, address newAddress);
   event ERC20Swept(address indexed token, address payee, uint256 amount);
-  event tokensSet(IERC20[] tokens);
-  event feesPaid(IERC20[] tokens, uint256[] amounts, uint256 timeCurser, bool half);
+  event TokensSet(IERC20[] tokens);
+  event FeesPaid(IERC20[] tokens, uint256[] amounts, uint256 timeCurser, bool half);
+  event MinAmountSet(uint256 minAmount);
 
   address public  keeperRegistry;
-  uint256 public lastRunTimeCurser;
+  uint256 public LastRunTimeCurser;
   IERC20[] public managedTokens;
   bool public half;
   IFeeDistributor public feeDistributor;
   bytes constant emptyBytes  = bytes("");
-
+  // to minimize changes after review, using a single minamount of all tokens, next version should support min amounts per tokens
+  uint256 public MinAmount;
 
 
     /**
    * @param _keeperRegistry The address of the keeper registry contract
    * @param _feeDistributor The address of the veBAL fee distributor
    * @param _tokens A list of tokens to handle.
    */
-  constructor(address _keeperRegistry, address _feeDistributor, IERC20[] memory _tokens) ConfirmedOwner(msg.sender)  {
+  constructor(address _keeperRegistry, address _feeDistributor, IERC20[] memory _tokens, uint256 minAmount) ConfirmedOwner(msg.sender)  {
```

Comment:
Open question from review: check if fee distributor works with 0 balances and and pays tokens even though one approved is 0.

Thinking about this more, we may send the tokens in at different times.  Therefore we only want to run if both tokens are present.  @mikebmikeb also pointed out that someone could greif this very easily for almost no money and make us notice/do work and added in a very simple minAmount feature that is the same for all tokens but decimal adjusted.

At least 100 BAL and 100 bb-a-usd v3 may be an example usage.  We did this to avoid a bigger change after review.